tests/utils/test_image_utils.py

```diff
@@ -58,13 +58,13 @@ def test_conversion_image_to_array(self):
         array3 = feature_extractor.to_numpy_array(image, rescale=False)
         self.assertTrue(array3.dtype, np.uint8)
         self.assertEqual(array3.shape, (3, 16, 32))
-        self.assertTrue(np.array_equal(array1, array3.astype(np.float32) / 255.0))
+        self.assertTrue(np.array_equal(array1, array3.astype(np.float32) * (1 / 255.0)))
```

Comment:
This was changed to reflect the rescaling logic. The max difference between the arrays before this change was ~5e-8.